static/scss/partials/fx-bento.scss

```diff
@@ -165,6 +165,10 @@ a.fx-bento-app-link {
     background-position-x: -18px;
 }
 
+.fx-bento-app-link-span.fx-vpn::before {
+    background-position-x: -162.5px;
```

Comment:
If it's possible to adjust this from a `.5px` to a whole pixel, let's do so. This may be my old browser-compat quirks showing but half-pixels can render strangely sometimes.

Reply:
Got it, half pixel is removed.